### ISSUE TITLE
fluid.nmfmorph~ crashes if buffer~ is modified after assignment

### ISSUE THREAD

**Issue by jamesb93** [flucoma/flucoma-max#257]
The patch below allows recreation of the issue.

[bug.maxpat.zip](https://github.com/flucoma/flucoma-max/files/8865299/bug.maxpat.zip)

[Max-2022-06-08-230242.ips.zip](https://github.com/flucoma/flucoma-max/files/8865304/Max-2022-06-08-230242.ips.zip)

**Comment by weefuzzy** [comment]
Yeah, if it changes size or something, and audio's on then all bets are possibly off. 

Offhand, I'm not sure what one could do about that.
